ocp_resources/storage_profile.py

```diff
@@ -4,4 +4,33 @@
 
 
 class StorageProfile(Resource):
+    """
+    StorageProfile Object
+
+    Doc:
+    https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md
+    """
+
     api_group = Resource.ApiGroup.CDI_KUBEVIRT_IO
+
+    @property
+    def claim_property_sets(self):
+        return self.instance.status.get("claimPropertySets", {})
```

Comment:
why take it from `status` and  not `spec`? - applies to all places
`claimPropertySets` is a list not a dict

Reply:
Because for the common case the `Spec` will be empty

When you create storgeclass the storageprofile is created automatically and its status params generated according to the storageclass defined
Spec is mostly to override/modify some params, but it will be updated in the status as well

So getting the params from the status is the right approach

Reply:
`claimPropertySets is a list not a dict`


right, my bad (I gave example with dict instead of list)